syserol/figures/common.py

```diff
@@ -269,26 +269,82 @@ def buildFigure3(legends=True, heatmap=False):
         for aa in ax:
             aa.axis('equal')
     else:
-        ax, f = getSetup((8, 8), (1, 4))
-        
-        cbar_kws = {"orientation":"horizontal"}
+        f = plt.figure(figsize=(10, 15))
```

Comment:
It's generally easier to have wide figures rather than tall ones. Can you put this back to having a row of plots?